e2e/fragments/draftDirections.js

```diff
@@ -0,0 +1,34 @@
+const { I } = inject();
+
+module.exports = {
+  fields: function (party, index) {
+    return {
+      direction: {
+        title: `#${party}_${index}_directionType`,
+        description: `#${party}_${index}_directionText`,
+        dueDate: {
+          day: `#${party}_${index}_dateToBeCompletedBy-day`,
+          month: `#${party}_${index}_dateToBeCompletedBy-month`,
+          year: `#${party}_${index}_dateToBeCompletedBy-year`,
+          hour: `#${party}_${index}_dateToBeCompletedBy-hour`,
+          minute: `#${party}_${index}_dateToBeCompletedBy-minute`,
+          second: `#${party}_${index}_dateToBeCompletedBy-second`,
+        },
+      },
+    };
+  },
+
+  enterDate(party, direction, index = 0) {
```

Comment:
Passing while direction to a matched that requires only one property of a direction is bad idea for many reasons. Always pass the minimum needed. It will make methods more meaningful / easier to understand. Passing more than needed suggests that much more is needed / method does much more than it is in reality.